xml/ha_fencing.xml

```diff
@@ -446,56 +446,62 @@ hostlist</screen>
     <para>Kdump belongs to the <xref linkend="sec-ha-fencing-special"
     xrefstyle="select:title"/> and is in fact the opposite of a fencing device.
      The plug-in checks if a Kernel dump is in progress on a node. If so, it
-     returns true, and acts <emphasis>as if</emphasis> the node has been fenced.
+     returns true and acts <emphasis>as if</emphasis> the node has been fenced,
+     because the node will reboot after the Kdump is complete.
+     If not, it returns a failure and the next fencing device is triggered.
     </para>
     <para>
-     The Kdump plug-in must be used in concert with another, real &stonith;
-     device, for example, <literal>external/ipmi</literal>. For the fencing
-     mechanism to work properly, you must specify that Kdump is checked before
-     a real &stonith; device is triggered. Use <command>crm configure
-     fencing_topology</command> to specify the order of the fencing devices as
+     The Kdump plug-in must be used together with another, real &stonith;
+     device, for example, <literal>external/ipmi</literal>. It does
+     <emphasis>not</emphasis> work with SBD as the &stonith; device. For the fencing
+     mechanism to work properly, you must specify the order of the fencing devices
+     so that Kdump is checked before a real &stonith; device is triggered, as
      shown in the following procedure.
    </para>
    <procedure>
     <step>
     <para>
-     Use the <literal>stonith:fence_kdump</literal> resource agent (provided
-     by the package <package>fence-agents</package>)
-     to monitor all nodes with the Kdump function enabled. Find a
-     configuration example for the resource below:
+     Use the <literal>stonith:fence_kdump</literal> resource agent
+     to monitor all nodes with the Kdump function enabled. A
+     configuration example is shown below. For more information,
+     see <command>crm ra info stonith:fence_kdump</command>.
     </para>
 <screen>&prompt.root;<command>crm configure</command>
 &prompt.crm.conf;<command>primitive st-kdump stonith:fence_kdump \
-    params nodename="&node1; "\ </command><co xml:id="co-ha-fenc-kdump-nodename"/>
+    params nodename="&node1; "\ </command><co xml:id="co-ha-fence-kdump-nodename"/>
     <command>pcmk_host_check="static-list" \
```

Comment:
```suggestion
    pcmk_host_list="&node1;" \
    <command>pcmk_host_check="static-list" \
```

REF: bsc#1007422

pcmk_host_list is documented in 
https://clusterlabs.org/pacemaker/doc/2.1/Pacemaker_Explained/singlehtml/#special-instance-attributes-for-fencing-resources